spec/shoryuken/manager_spec.rb

```diff
@@ -8,6 +8,15 @@
     Shoryuken::Manager.new(condvar)
   end
 
+  describe 'Invalid concurrency setting' do
+    it 'raises ArgumentError if concurrency is not positive number' do
+      Shoryuken.options[:concurrency] = -1
+      expect{Shoryuken::Manager.new(nil)}
```

Comment:
Space missing to the left of {.<br>Space missing inside {.<br>Space missing inside }.